migrationmanager/migrators/traces/migrations/000028_drop_deprecated_columns.up.sql

```diff
@@ -0,0 +1,132 @@
+DROP TABLE IF EXISTS signoz_traces.durationSortMV ON CLUSTER {{.SIGNOZ_CLUSTER}};
+DROP VIEW IF EXISTS signoz_traces.dependency_graph_minutes_db_calls_mv ON CLUSTER {{.SIGNOZ_CLUSTER}};
+DROP VIEW IF EXISTS signoz_traces.dependency_graph_minutes_messaging_calls_mv ON CLUSTER {{.SIGNOZ_CLUSTER}};
+DROP VIEW IF EXISTS signoz_traces.dependency_graph_minutes_db_calls_mv_v2 ON CLUSTER {{.SIGNOZ_CLUSTER}};
+DROP VIEW IF EXISTS signoz_traces.dependency_graph_minutes_messaging_calls_mv_v2 ON CLUSTER {{.SIGNOZ_CLUSTER}};
+
+CREATE MATERIALIZED VIEW IF NOT EXISTS signoz_traces.dependency_graph_minutes_db_calls_mv ON CLUSTER {{.SIGNOZ_CLUSTER}}
+TO signoz_traces.dependency_graph_minutes AS
+SELECT
+    serviceName as src,
+    dbSystem as dest,
+    quantilesState(0.5, 0.75, 0.9, 0.95, 0.99)(toFloat64(durationNano)) as duration_quantiles_state,
+    countIf(statusCode=2) as error_count,
+    count(*) as total_count,
+    toStartOfMinute(timestamp) as timestamp
+FROM signoz_traces.signoz_index_v2
+WHERE dest != '' and kind != 2
+GROUP BY timestamp, src, dest;
+
+CREATE MATERIALIZED VIEW IF NOT EXISTS signoz_traces.dependency_graph_minutes_messaging_calls_mv ON CLUSTER {{.SIGNOZ_CLUSTER}}
```

Comment:
We only need v2.

Reply:
Since first version tables were not dropped, it was giving errors.

Reply:
Can you share the error?

Reply:
```
{"level":"error","timestamp":"2024-05-24T18:41:38.691+0530","caller":"migrationmanager/manager.go:81","msg":"Failed to run migrations for migrator","component":"migrationmanager","migrator":"traces","error":"clickhouse migrate failed to run, error: migration failed in line 0: \n\nALTER TABLE signoz_traces.signoz_index_v2 ON CLUSTER cluster \nDROP INDEX idx_httpCode,\nDROP INDEX idx_tagMapKeys,\nDROP INDEX idx_tagMapValues,\nDROP COLUMN IF EXISTS tagMap,\nDROP COLUMN IF EXISTS gRPCCode,\nDROP COLUMN IF EXISTS gRPCMethod,\nDROP COLUMN IF EXISTS httpCode,\nDROP COLUMN IF EXISTS component; (details: code: 524, message: There was an error on [clickhouse:9000]: Code: 524. DB::Exception: Trying to ALTER DROP column tagMap which is referenced by materialized view ['dependency_graph_minutes_db_calls_mv']. (ALTER_OF_COLUMN_IS_FORBIDDEN) (version 24.1.2.5 (official build)))","stacktrace":"github.com/SigNoz/signoz-otel-collector/migrationmanager.(*MigrationManager).Migrate\n\t/Users/makeavish/signoz/signoz-otel-collector/migrationmanager/manager.go:81\nmain.main\n\t/Users/makeavish/signoz/signoz-otel-collector/cmd/signozschemamigrator/migrate.go:126\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:271"}
{"level":"fatal","timestamp":"2024-05-24T18:41:38.691+0530","caller":"signozschemamigrator/migrate.go:128","msg":"Failed to run migrations","component":"migrate cli","error":"clickhouse migrate failed to run, error: migration failed in line 0: \n\nALTER TABLE signoz_traces.signoz_index_v2 ON CLUSTER cluster \nDROP INDEX idx_httpCode,\nDROP INDEX idx_tagMapKeys,\nDROP INDEX idx_tagMapValues,\nDROP COLUMN IF EXISTS tagMap,\nDROP COLUMN IF EXISTS gRPCCode,\nDROP COLUMN IF EXISTS gRPCMethod,\nDROP COLUMN IF EXISTS httpCode,\nDROP COLUMN IF EXISTS component; (details: code: 524, message: There was an error on [clickhouse:9000]: Code: 524. DB::Exception: Trying to ALTER DROP column tagMap which is referenced by materialized view ['dependency_graph_minutes_db_calls_mv']. (ALTER_OF_COLUMN_IS_FORBIDDEN) (version 24.1.2.5 (official build)))","stacktrace":"main.main\n\t/Users/makeavish/signoz/signoz-otel-collector/cmd/signozschemamigrator/migrate.go:128\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:271"}
```